api/lib/domain/usecases/create-organizations-with-tags-and-target-profiles.js

```diff
@@ -170,7 +185,9 @@ function _mapOrganizationsData(organizations) {
       },
       emailInvitations: organization.emailInvitations,
       tags: organization.tags.split(SEPARATOR),
-      targetProfiles: organization.targetProfiles.split(SEPARATOR).filter((targetProfile) => !!targetProfile.trim()),
+      targetProfiles: lodash.isEmpty(organization.targetProfiles)
```

Comment:
Lodash étant destructuré à la ligne 18.

```suggestion
      targetProfiles: isEmpty(organization.targetProfiles)
```

Je suppose que cette modification est dû à un test qui ne passait pas parce que la propriété `targetProfiles` n'était pas défini ?

Reply:
done